gqlalchemy/reference/transformations/importing/import.md

```diff
@@ -0,0 +1,101 @@
+---
+sidebar_label: import
+title: gqlalchemy.transformations.importing.import
+---
+
+# Abstract
+
+GQLAlchemy supports importing data from various sources. This includes import from graph objects stored in NetworkX, DGL and PyG format but also importing from various other table sources like S3, Azure Blob storage... Files stored in local file system could be of any of the CSV, Parquet, ORC or Feather format and GQLAlchemy can parse such data and import it to the Memgraph database.
+
+## Importer
+
+Abstract importer that doesn't implement any strategy for translating graph but provides interface that other objects can implement. Each `Importer` has a reference to the specific translator object which is used for translating graph of specific type into Memgraph data. This translator does all the dirty stuff so check out docs about `DGLTranslator, PyGTranslator and NxTranslator` to find out implementation details.
+
+
+## GraphImporter
+
+`GraphImporter` is one such object that implements `Importer`. It is used an entry class for all importing stuff where the original object is in the graph shape already. Currently, importing from `NetworkX, DGL and PyG` is supported.
```

Comment:
please avoid using the word "stuff"
the 2nd sentence doesn't make any sense
Currently, WHAT supports importing from .... please code format individual wrods/names, not a group of strings